zuul.d/nodesets.yaml

```diff
@@ -24,12 +24,24 @@
       - name: ubuntu-xenial-vexxhost
         label: ubuntu-xenial-vexxhost
 
+- nodeset:
+    name: ubuntu-xenial-vxh-arm64
```

Comment:
Only `ubuntu-xenial-arm64` is available now in OpenLab, please update nodeset name and remove the following two ARM nodeset, thx.